src/css/styles.css

```diff
@@ -57,12 +57,28 @@ p {
   font-weight: var(--t-font-weight-regular);
 }
 
+/*
+ * Components
+ * 1. Avatar
+ */
+
+/* Avatar */
+.avatar {
+  display: inline-block;
+  vertical-align: middle;
+  font-size: var(--t-font-size-md);
+  block-size: 2.5em;
+  inline-size: 2.5em;
```

Comment:
you can give this size in avatar-lg , avaatr-md classes ... your avatar class should be generic

Reply:
Here I have a generic class for avatar and I'm using utility class to modify it..size and border radius

Reply:
ok